webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPage.js

```diff
@@ -7,49 +7,47 @@ import { translate as __ } from '../../../common/I18n';
 import PageLayout from '../../common/PageLayout/PageLayout';
 import AuditsTable from './components/AuditsTable';
 import { AUDITS_SEARCH_PROPS, AUDITS_MANUAL_URL } from '../constants';
+import { useForemanVersion } from '../../../Root/Context/ForemanContext';
 
 const AuditsPage = ({
   searchQuery,
   fetchAndPush,
-  version,
   isLoading,
   hasData,
   ...props
-}) => (
-  <PageLayout
-    header={__('Audits')}
-    searchable
-    searchProps={AUDITS_SEARCH_PROPS}
-    searchQuery={searchQuery}
-    isLoading={isLoading && hasData}
-    onSearch={search => fetchAndPush({ searchQuery: search, page: 1 })}
-    onBookmarkClick={search => fetchAndPush({ searchQuery: search, page: 1 })}
-    toolbarButtons={
-      <Button href={AUDITS_MANUAL_URL(version)} className="btn-docs">
-        <Icon type="pf" name="help" />
-        {__(' Documentation')}
-      </Button>
-    }
-  >
-    <AuditsTable
-      fetchAndPush={fetchAndPush}
-      isLoading={isLoading}
-      hasData={hasData}
-      {...props}
-    />
-  </PageLayout>
-);
+}) => {
+  const foremanVersion = useForemanVersion();
+  return (
+    <PageLayout
+      header={__('Audits')}
+      searchable
+      searchProps={AUDITS_SEARCH_PROPS}
+      searchQuery={searchQuery}
+      isLoading={isLoading && hasData}
+      onSearch={search => fetchAndPush({ searchQuery: search, page: 1 })}
+      onBookmarkClick={search => fetchAndPush({ searchQuery: search, page: 1 })}
+      toolbarButtons={
+        <Button href={AUDITS_MANUAL_URL(foremanVersion)} className="btn-docs">
```

Comment:
Not directly related, but we have `documentation_url` that i think should be used instead, which might make the `foremanVersion` redundant?